docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -4,11 +4,19 @@
 Distributing collections
 ************************
 
-You can distribute your collections by publishing them on a distribution server. Distribution servers include Ansible Galaxy, Red Hat Automation Hub, and privately hosted Automation Hub instances. You can publish any collection to Ansible Galaxy and/or to a privately hosted Automation Hub instance. If your collection is certified by Red Hat, you can publish it to the Red Hat Automation Hub.
+A collection is a distribution format for Ansible content. A typical collection contains modules and other plugins that address a set of related use cases. For example, a collection might automate administering a particular database. A collection can also contain roles and playbooks.
 
-Distributing collections involves three major steps:
-#. Configuring your distribution server(s)
-#. Building your collection artifact
+To distribute your collection and allow others to use it, you can publish your collection on one or more distribution servers. Distribution servers include:
+
+* Ansible Galaxy (accepts all collections)
+* Red Hat Automation Hub (accepts only collections certified by Red Hat)
+* privately hosted Automation Hub instances (accepts collections authorized by the owners)
+
+Distributing collections involves four major steps:
+
+#. Initial configuration of your distribution server or servers
```

Comment:
The #. was showing up as #. in the actual documentation. Should these not be replaced with a numbered list?

Reply:
The `#.` is the correct syntax for building numbered lists in Sphinx. In the original this was showing up as `#.` because the doc didn't have a blank line above the numbered list. This PR fixes the problem by adding the blank line, see http://docs.testing.ansible.com/ansible-core/devel/dev_guide/developing_collections_distributing.html.